eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -130,10 +131,23 @@ def fitJWST(eventlabel, s4_meta=None):
                 params = p.Parameters(param_file=meta.fit_par)
 
                 # Subtract off the zeroth time value to avoid floating point precision problems when fitting for t0
-                t_offset = int(np.floor(meta.bjdtdb[0]))
-                t_mjdtdb = meta.bjdtdb - t_offset
-                params.t0.value -= t_offset
-
+                offset = int(np.floor(meta.time[0]))
```

Comment:
This code will get messy as we implement more and more models with time offsets, like phase curves. Also, updating params.dict could lead to confusion when users compare inputs vs outputs. Maybe it's better to educate users on not using the full BJD in their initial parameters file?

Reply:
Your thinking is that we require the inputs in the parameter ecf be in MBJD (BJD-2400000.5) and then only make any adjustments to the time array in Stage 5? Or even make any time adjustments in Stage 3 only?

Reply:
Thinking about max flexibility, we cold let users define the time offset in the ecf file. If none is given, then use the current definition. We don't want to tie ourselves to choices in S3 that don't impact the reduction. That'll just lead to frustration (speaking from experience). Happy to talk more about this.

Reply:
To me, the danger with your suggested method is that everybody needs to be sure to subtract the time_offset from all of the time-based parameters in their parameter ecf, and not doing so will lead to significant issues without any hints as to what could have gone wrong to the user. I'll proceed with your suggested method, but we should think carefully about how we want to protect users from confusion